src/components/core/Text/Text.tsx

```diff
@@ -50,7 +50,7 @@ export const Subdisplay = styled(H2)`
   letter-spacing: 0px;
 
   @media only screen and ${breakpoints.tablet} {
-    font-size: 40px;
+    font-size: 48px;
```

Comment:
`Subdisplay` is only used by the Gallery username header and the ErrorBoundary message so changing the font size here should be ok